worker/httpserver/tls.go

```diff
@@ -31,23 +29,19 @@ func aggregateSNIGetter(getters ...SNIGetterFunc) SNIGetterFunc {
 			}
 		}
 		return nil, lastErr
-	})
+	}
 }
 
 // NewTLSConfig returns the TLS configuration for the HTTP server to use
 // based on controller configuration stored in the state database.
```

Comment:
I think we need to slow down here and give some attention to comments. This one no longer makes sense, and is now a potential source of confusion for readers.